### ISSUE TITLE
columnInsights.displayOnOpen setting not respected

### ISSUE THREAD

**Issue by mdj-uk** [microsoft/vscode-data-wrangler#338]
Type: <b>Bug</b>

# Behaviour

## Expected vs. Actual

With `dataWrangler.columnInsights.displayOnOpen` set to false, I would expect the data wrangler columns insights side panel _not_ to open when I choose to view a variable in 'data viewer' from Jupyter's variables pane.

But it always does open.

## Steps to reproduce:

1. run some python code in a jupyter notebook or interactive window
2. From variables pane, choose to open a variable in data viewer

# Diagnostic data

-   Jupyter extension version: 2024.10.0
-   Python extension version: 2024.18.1
-   .NET Install Tool for Extension Authors extension version: Not installed
-   Python package dependencies:

```json
{
  "installed": {
    "pandas": "2.2.3"
  },
  "required": {
    "pandas": "1.2.0"
  },
  "unsatisfied": []
}
```

-   Entrypoint: JupyterDataViewer
-   Active mode: dataViewer

Extension version: 1.12.1
VS Code version: Code 1.95.1 (Universal) (65edc4939843c90c34d61f4ce11704f09d3e5cb6, 2024-10-31T05:14:54.222Z)
OS version: Darwin x64 23.2.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz (4 x 1600)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|16, 110, 78|
|Memory (System)|16.00GB (0.12GB free)|
|Process Argv|--crash-reporter-id 36a94c38-386a-460d-9972-6a3c8031c95d|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
9c06g630:31013171
dvdeprecation:31068756
dwnewjupytercf:31046870
impr_priority:31102340
nativerepl1:31139838
refactort:31108082
pythonrstrctxt:31112756
cf971741:31144450
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013
h409b430:31179529

```

</details>

https://github.com/user-attachments/assets/de3eb08d-c2f4-49bc-a3f6-cddf1e6e1452

**Comment by mdj-uk** [comment]
Closing this as I've misunderstood what the column insights view is. Though it's still pretty annoying that there seems to be no way to prevent the side panel appearing as soon as I load some data.

**Comment by pwang347** [comment]
Hi @mdj-uk, thanks for opening this issue! Just wanted to let you know that this setting is now available in pre-release version 1.13.3. You can check out this comment for more details: https://github.com/microsoft/vscode-data-wrangler/issues/96#issuecomment-2487056292

Hope this helps.
